nservicebus/sagas/index.md

```diff
@@ -193,5 +193,6 @@ Saga state read just before `Handle` is invoked and written immediately after `H
 - The write will be an ADD if the read didn't return data or an UPDATE if the read did return data.
 - If the read did not return data and during the invoke the saga is completed no write will occur
 - How state remains consistent depends the persister implementation
+- Saga state reads/writes happen after all pipeline stages
```

Comment:
That's not correct since we do it in the `IInvokeHandlerContext`, did you mean "handlers for the given message"?

Reply:
@andreasohlund I've reworded the sentence. What I wanted to make clear is that saga state is not read or written at/in a stage, but that the state is read/written as part of handler invocation. Basically, that state cannot be intercepted somewhere due to this design.